ndc-reference/tests/query/group_by_with_star_count/expected.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "groups": [
+      {
+        "dimensions": [
+          1
+        ],
+        "aggregates": {
+          "article_count": 1
+        }
+      },
```

Comment:
I'm wondering why this isn't:

```json
      {
        "author_id": 1,
        "article_count": 1
      },
```

?

Reply:
Because dimensions don't have names, and if they did, they might overlap.

Reply:
Could you give an example please? My context for understanding this is sql, I understand that dimensions are effectively group by keys? And group by keys are named. Am I missing something?

Reply:
Also, am I right in understanding that we have a separation between dimensions and aggregates in the result to mimic the request structure?

Reply:
The dimensions/aggregates split comes from multidimensional analysis. It's a bit awkward in SQL because everything is combined into one expression language, but then not all expressions are equal - you can't compute all expressions per-group, for example. In SQL, everything would be a named member of the target list, but in general, you just have an ordered list of dimensions to partition the data by. 

Yes, the response is the same as the request.